app/templates/components/modals/tax-info-modal.hbs

```diff
@@ -21,20 +21,20 @@
     </div>
     <div class="field">
       <label class="required">{{t 'Tax name'}}</label>
-      <Input @type="text" @id="tax_name" @value={{this.tax.name}} />
+      <Input @type="text"  @id="tax_name" @value={{this.tax.name}} />
```

Comment:
Extra space. Ensure that no changes other than in country field are present in the files changed

Reply:
@iamareebjamal Sorry sir didn't notice that. Removed the extra spaces